### PR TITLE
Fix (v)maxss on non-AFP platforms

### PR DESCRIPTION
**Pull request by alyssarosenzweig**
Fixes miscompiles in the vulkan cts and probably more.

### CODE REVIEW

File: unittests/ASM/FEX_bugs/MinMaxNaN.asm [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+%endmacro
+
+%macro max 3
+  single_case max, %1, %2, %3
+%endmacro
+
+zero equ 0x00000000
+negzero equ 0x80000000
+qnan equ 0x7fc00000
+snan equ 0x7fc00000
```
Comment (Sonicadvance1) [inline unittests/ASM/FEX_bugs/MinMaxNaN.asm:61]: Is snan and qnan expected to be the same here?

Comment (alyssarosenzweig) [reply unittests/ASM/FEX_bugs/MinMaxNaN.asm:61]: doh. No, of course not, copypaste fail

Comment (alyssarosenzweig) [reply unittests/ASM/FEX_bugs/MinMaxNaN.asm]: fixed, thanks!

**Review by Sonicadvance1 — APPROVED** [review]
nice catch!

**Pull request merged**